boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350.dts

```diff
@@ -26,7 +26,8 @@
 		zephyr,console = &uart1;
 		zephyr,shell-uart = &uart1;
 		zephyr,sram = &dram;
-		zephyr,flash = &flash0;
+		zephyr,flash = &mx25u16;
```

Comment:
Commit message is confusing - is flash or i2c driver being supported?

Reply:
Thanks. I've edited the commit message.